### PR TITLE
Ingestion for the vulnerabilities

### PR DESCRIPTION
**Pull request by mihaimaruseac**

### CODE REVIEW

**Comment by mihaimaruseac** [comment]
(if reviewing commit by commit, first 4 are not from this PR, bad conflict resolution where a delta was pushed to the wrong commit and removed later)

File: pkg/assembler/backends/neo4j/cve.go [hunk 1]
```diff
@@ -162,29 +161,27 @@ func (c *neo4jClient) Cve(ctx context.Context, cveSpec *model.CVESpec) ([]*model
...
-			var firstMatch bool = true
-			queryValues := map[string]any{}
-
-			sb.WriteString("MATCH (n:Cve)-[:CveIsYear]->(cveYear:CveYear)-[:CveHasID]->(cveID:CveID)")
+	var sb strings.Builder
+	var firstMatch bool = true
+	queryValues := map[string]any{}
 
-			if cveSpec.Year != nil {
+	sb.WriteString("MATCH (n:Cve)-[:CveIsYear]->(cveYear:CveYear)-[:CveHasID]->(cveID:CveID)")
```
Comment (pxp928) [inline pkg/assembler/backends/neo4j/cve.go:168]: can we change the variables to match both queries and ingest for consistency?

**Review by pxp928 — APPROVED** [review]
LGTM

File: pkg/assembler/backends/neo4j/cve.go (hunk 1 above)
Comment (mihaimaruseac) [reply pkg/assembler/backends/neo4j/cve.go:168]: Sure, changed the query vars

**Review by lumjjb — APPROVED** [review]

**Pull request merged**